notebooks/QA_evaluation_metrics_demo.ipynb

```diff
@@ -0,0 +1,1536 @@
+{
```

Comment:
<p>What is that deeper understanding?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/foundation-models-for-documentation/pull/41/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>